other/allowed_pod_priorities/values.yaml

```diff
@@ -4,13 +4,12 @@ policies:
       - name: validate-pod-priority-pods
         values:
           podprioritydict.data.default: "[\"high-priority\", \"moderate-priority\", \"low-priority\"]"
-
+          request.namespace: default
```

Comment:
LGTM - can you verify with the kyverno CLI?

Reply:
Verified good:

```sh
Executing allowed-podpriorities...
applying 1 policy to 3 resources...

│───│───────────────────────│───────────────────────────────│─────────────────────────────│────────│
│ # │ POLICY                │ RULE                          │ RESOURCE                    │ RESULT │
│───│───────────────────────│───────────────────────────────│─────────────────────────────│────────│
│ 1 │ allowed-podpriorities │ validate-pod-priority-pods    │ default/Pod/myapp-pod       │ Pass   │
│ 2 │ allowed-podpriorities │ validate-pod-priority         │ default/Deployment/mydeploy │ Pass   │
│ 3 │ allowed-podpriorities │ validate-pod-priority-cronjob │ default/CronJob/hello       │ Pass   │
│───│───────────────────────│───────────────────────────────│─────────────────────────────│────────│
```

Reply:
Nice, thanks!